maestrowf/interfaces/script/lsfscriptadapter.py

```diff
@@ -155,23 +159,21 @@ def get_parallelize_command(self, procs, nodes=None, **kwargs):
         :returns: A string of the parallelize command configured using nodes
                   and procs.
         """
-        args = [self._cmd_flags["cmd"]]
 
-        if nodes:
-            _nodes = nodes
-            args += [
-                self._cmd_flags["nodes"],
-                str(nodes)
-            ]
-        else:
-            _nodes = 1
-
-        # Compute the number of CPUs per node (rs)
-        _procs = int(procs)/int(_nodes)
+        
+        args = [self._cmd_flags["cmd"]]
 
         # Processors segment
         args += [
-            self._cmd_flags["ntasks"].format(procs=_procs)
+            self._cmd_flags["ntasks"],
+            procs
```

Comment:
This item looks like it needs to be cases as a string -- it's making it to the end and the concatenation fails.

```suggestion
            str(procs)
```

Reply:
Got that; sorry about that one.